app/decorators/news_tematica/news_tematica_decorator.rb

```diff
@@ -79,11 +75,11 @@ def evita_error_divbyzero(dato)
     end
 
     def titulo_generico
-      general? ? "News de #{ESTA_WEB}" : "News temática de #{tematica.nombre}"
+      general? ? "News de #{ESTA_WEB}" : "News temática de #{suscribible.nombre}"
     end
 
     def titulo_de_foro
-      general? ? "Foros" : "Foro de #{tematica.nombre}"
+      general? ? "Foros" : "Foro de #{suscribible.nombre}"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.